### PR TITLE
Add keploy via

### PR DESCRIPTION
**Pull request by Sonichigo**
## Description
This PR attempts to add support for downloading and installing the `Keploy` for supported platforms (Linux, MacOS, Windows).

## Motivation and Context

- [x] I have raised an issue to propose this change, which has been given a label of `design/approved` by a maintainer ([required](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md))

## How Has This Been Tested?

I have tested on MacOs and WSL (for combined testing for windows and linux)
```
## wsl
go run . get keploy 

## macos
go run . get keploy --arch arm64
```

If updating or adding a new CLI to `arkade get`, run:

```
go build && ./hack/test-tool.sh keploy
```

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Documentation

- [x] I have updated the list of tools in README.md if (required) with `./arkade get --format markdown`
- [x] I have updated the list of apps in README.md if (required) with `./arkade install --help`

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I've read the [CONTRIBUTION](https://github.com/alexellis/arkade/blob/master/CONTRIBUTING.md) guide
- [x] I have signed-off my commits with `git commit -s`

- [x] I have tested this on arm, or have added code to prevent deployment

### CODE REVIEW

**Comment by alexellis** [comment]
Hi @Sonichigo 

This PR is for a product which you're employed to promote as a developer advocate.

If merged, would you update the website to include arkade as the installation method?

image

**Comment by Sonichigo** [comment]
Yes we can add on our installation guide that's the plan : - https://keploy.io/docs/server/installation/ . On the main landing page, we are removing the above section with https://github.com/keploy/website/pull/42 section . We initially had kept the get started for just people to try in simple steps.

**Comment by alexellis** [comment]
OK, so you'll need to read the PR template/description carefully and follow the instructions. Once done, let us know.

**Comment by Sonichigo** [comment]
hey @alexellis i'm done please review and let me know if there's any more changes need to be done

**Comment by alexellis** [comment]
Let me ping @Jasstkn / @Shikachuu to have a look a this for you.

File: pkg/get/get_test.go
Comment (Shikachuu) [inline pkg/get/get_test.go:7749]: I'm kinda against the idea of using an alpha version here, could result some false negatives in the future since there is a higher chance that overtime they'll get rid of the alpha and beta releases, but keep the normal ones.

File: pkg/get/get_test.go
Comment (Shikachuu) [inline pkg/get/get_test.go:7775]: what about intel based macs? (`arch64bit` + `darwin`)

**Review by Shikachuu — CHANGES_REQUESTED** [review]

File: pkg/get/get_test.go
Comment (Sonichigo) [reply pkg/get/get_test.go:7775]: not yet supported, it's in long term roadmap

File: pkg/get/get_test.go
Comment (Sonichigo) [reply pkg/get/get_test.go:7749]: right now, there are only alpha versions and only the versioning would change and not tag's. alpha-19 is outdated with alpha-22. I noticed in above test as well the versions were outdated so i went with most recent version at that time.

**Comment by Jasstkn** [comment]
@Sonichigo hi. could you follow instructions [after the table here](https://github.com/alexellis/arkade?tab=readme-ov-file#catalog-of-clis) and regenerate the table in the README? Overall, changes LGTM 🚀

**Comment by Sonichigo** [comment]
@Jasstkn done. can you take a look.

**Comment by Shikachuu** [comment]
I'm sorta kinda still not like the alpha tags but let's roll with this and we'll see.

However can you please do a squash to a single signed commit before we merge this?

Thanks!

**Comment by Sonichigo** [comment]
> I'm sorta kinda still not like the alpha tags but let's roll with this and we'll see.
> 
> However can you please do a squash to a single signed commit before we merge this?
> 
> Thanks!

I'll keep an eye out with the relase without any alpha or beta tags, once it is released I'll update the version in get file. I think the squash and merge is available from github web interface 
image

**Comment by rgee0** [comment]
> 
> I'll keep an eye out with the relase without any alpha or beta tags, once it is released I'll update the version in get file. I think the squash and merge is available from github web interface 

If you can squash locally and force push then Derek will remove the `no-dco` label

The facility you've provided a screen shot of is only available to the maintainers and they're unlikely to take a further look at PRs where the `no-dco` label is showing.

**Comment by Sonichigo** [comment]
@Shikachuu @rgee0 @Jasstkn closing this pr since lot of commit history is polluated please take a look at this one:- https://github.com/alexellis/arkade/pull/1104

**Pull request closed without merging**
